types/test/mount.spec.ts

```diff
@@ -28,6 +28,7 @@ mount<ClassComponent>(ClassComponent, {
         default: ClassComponent,
         foo: [ClassComponent],
     },
+    renderDefaultSlot: true,
```

Comment:
Ah, there isn't a test for shallow. You'll need to import shallow in this line - https://github.com/disitec/avoriaz/blob/8d01a6a631f5ebe091b42c70d753b6ad3f14b226/types/test/mount.spec.ts#L3

And then write a test similar to the one with mount. In other words, just call shallow with the option and make sure the TS compiler doesn't throw an error.